tutorials/vscode.md

```diff
@@ -11,4 +11,60 @@
 This page contains **'community resources' contributed by students**.
 </box>
 
-Coming soon ...
\ No newline at end of file
+## Importing a Gradle project
+
+<box type="tip" seamless>
+
+If the project comes with Gradle support, you will see a build.gradle file in your project root.
+</box>
+
+<box type="warning" seamless>
+
+Install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) extension to enable support for Java extensions.  
+Go to `Extensions` → Search `Extension Pack for Java` → Install.
+</box>
+
+1. Open Visual Studio Code.
+
+2. If you are in the welcome screen, Click Open. Otherwise, click File -> Open.
+   i. Select the project directory, and click Open.
```

Comment:
I understand the intended step-by-step approach to indicate opening a file. However, when I first read it, I was slightly confused with all the 'Open's. Perhaps for just opening the project, we could tell the reader to "Open your project via Open Folder or File -> Open. Then select your project folder and click open." rather than have a (i) point for it.